include/dem/particle_particle_contact_force.h

```diff
@@ -556,7 +557,7 @@ class ParticleParticleContactForce
     double coulomb_threshold =
```

Comment:
```suggestion
    const double coulomb_threshold =
```

Reply:
Will do in the other PR, this is not a big thing